pyfr/integrators/dual/controllers.py

```diff
@@ -68,10 +70,13 @@ def advance_to(self, t):
                 # Activate convergence monitoring after pseudo-niters-min
                 if i >= self._minniters - 1:
                     # Subtract the current solution from the previous solution
-                    self._add(-1.0/dtau, self._idxprev, 1.0/dtau, self._idxcurr)
```

Comment:
No, the break condition is the same and just reformulated, residual=resid tol. Before we computed the residual with this add kernel and then used the errest kernel to relate the residual to the tolerances and the break criterion was normalised so that were comparing to unity.  This was unecessarily confusing.  In addition, we want to write to file the absolute residuals, not the ones that are normalised with the tolerances.